src/item.cpp

```diff
@@ -2237,14 +2268,20 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
-    if( t == nullptr ) {
+    float pockets = 1;
+    auto t = find_armor_data();
+    if( t == nullptr )
         return 0;
-    }
+
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
-}
+    int result = (static_cast<int> (static_cast<unsigned int>( t->storage ) ) );
 
+    if (item::item_tags.count("pocketed") > 0){
+        pockets = (volume() * get_coverage()) / 100;
+        return result + pockets;
```

Comment:
Problem: this also applies to cargo pants.  Pocketed Kevlar'd army pants compare favorably to survivor cargo pants, except for a lack of environmental protection.

I'm thinking cap: if the clothing already has more than half its volume in storage, pockets is cut by half.  There simply isn't room for more storage.